### PR TITLE
Better explain how to enable the validation component and its annotations

### LINKED ISSUE #7797: Validation not enabled by default
In `validation.rst` is written:

> Configuration
> 
> The Symfony validator is enabled by default, but you must explicitly enable annotations if you're using the annotation method to specify your constraints:

But actually, in my fresh Symfony installations (with installer 1.5.9) the annotation **is** enabled by default like:

    validation:      { enable_annotations: true }

But I have to explicitly enable when I dont want to use annotations (so exactly the opposite to what is documented in validation.rst):

    validation:      { enabled: true }

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7797.

### CODE REVIEW

File: validation.rst [hunk 1]
```diff
@@ -212,8 +212,43 @@ Configuration
...
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'validation' => array(
+                'enabled' => true,
+            ),
+        ));
+
+Besides, if you plan to use annotations to configure validation (as recommended
+by the official Symfony Best Practices), replace the previous configuration by
```
Comment (HeahDude) [inline validation.rst]: Should we add some link?

Comment (javiereguiluz) [inline validation.rst]: I cannot find any reference to this in the Best Practices. So it's not a best practice but just my imagination inventing things 😁

**Review by HeahDude — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**